common/src/test/resources/testPayment.properties

```diff
@@ -0,0 +1,10 @@
+# needed for convinience to make parsing easier and more independed of parameters from outside
+methods.interface=test
+
+# comma separated list of available payment method IDs
+methods.available=test-methodid
+
+# config for each method
+methods.test-methodid.method=test-methodid
```

Comment:
@lauraluiz I'll consider that for the implementation. I added the config class requirement to the issues #11